UI changed to CreatePlayerView

jtmcginnis opened the pull request:
StoryGame is now the master UI
CreatePlayerView and MapView are accessed from StoryGame

doctor-g commented:
"CreatePlayerView" strikes me as problematic because it can be read in two ways, including as a verb phrase "create the player view". If it's the view for player creation, how about the unambiguous `PlayerCreationView`?

jtmcginnis committed:
UI changed to CreatePlayerView
StoryGame is now the master UI
CreatePlayerView and MapView are accessed from StoryGame

jtmcginnis committed:
UI changed to CreatePlayerView
StoryGame is now the master UI
CreatePlayerView and MapView are accessed from StoryGame
Player Constructor is Fixed
